### ISSUE TITLE
On MacOS the project can't be loaded

### ISSUE THREAD

**Issue by bashcole** [EdmarDev/Wind-Up-Racer#2]
Since I'm slowly learning Godot I was excited to see how you implement the game mechanics.. but unfortunately the project doesn't load and the Godot application crashes. I tried multiple times after restart to no success

**Comment by EdmarDev** [comment]
I don't have a mac so I can't really test anything, did you try making a new project and just importing the files? Are you using the latest godot version? (I guess I should have mentioned this somewhere, this was made on 3.2.2 stable)

**Comment by bashcole** [comment]
Yeah, I'm using the latest version. I created a new project and imported all of the assets, then replace the project.godot file (because of input map settings) and it worked!

What is interesting that still the downloaded project crashes Godot and also If a move the newly created project and try to open it.. Godot crashes. Probably because it's re-importing the assets and the import dialog will reach road_texture.png or arena.obj before Godot crashes. Could be a problem with the MacOS version.. file permission issues, but this happens for the first time I have opened a lot of 2D/3D games published on GitHub/GitLab
